supreme_court_predictions/statistics/descriptives.py

```diff
@@ -267,6 +235,49 @@ def get_voters_desc(self):
 
         return voter_stats
 
+    def get_cases_desc(self):
+        """
+        Calculates descriptive statistics for the cases DataFrame.
+
+        :return DataFrame of descriptive statistics.
+        """
+        # Load data
+        cases = pd.read_csv(self.local_path + "cases_df.csv")
+
+        # Get dataframe of winning side descriptive stats
+        winning_sides = [
+            "for petitioner",
+            "for respondent",
+            "unclear",
+            "unavailable",
+        ]
+
+        cases_stats = self.get_count_desc(cases, ["win_side"])
+
+        cases_stats.index = self.fix_indices("win side", winning_sides)
+
+        # get counts
```

Comment:
You capitalize most of your comments, could you do that for the new ones you've added?